### DISCUSSION TITLE
can someone scrape an account for me

### DISCUSSION THREAD

**Discussion by lonewo1f1234** [UltimaHoarder/UltimaScraper#1416] [Q&A]
can someone scrape an account for me?

**Comment by darkstarlogin** [comment]
Do you mean scrape an OF creator's page for you? They'd have to be subscribed (or have your login credentials). Or are you trying to test something?

**Reply by lonewo1f1234** [reply]
Scrape an onlyfans creator page for me

**Reply by darkstarlogin** [reply]
Is it someone you subscribe to? If so, I'm sure someone would be happy to help you learn to use the script. Otherwise, you may be in the wrong place.

**Reply by lonewo1f1234** [reply]
Yea that would be great idk how to use the script  if someone could show me that would be great

**Comment by lucky1luc** [comment]
sure I can, give me username, password and I will do that for you. Are you kidding me????
